lib/logidze.rb

```diff
@@ -68,4 +70,5 @@ def with_logidze_setting(name, value)
   self.ignore_log_data_by_default = false
   self.return_self_if_log_data_is_empty = true
   self.on_pending_upgrade = :ignore
+  self.sort_triggers_by_name = :false
```

Comment:
I think, RuboCop is right and it should be `false`)

```suggestion
  self.sort_triggers_by_name = false
```